en/docs/learn/self-registration-and-account-confirmation.md

```diff
@@ -265,7 +242,21 @@ Next, you can try out self-registration.
         You can edit and customize the email template. For more information on
         how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails).
+
+!!! Note
+    If you wish to send a account unlocked email upon account confirmation, please do the following 
+    configurations.
+    
+    1. Navigate to **Resident IDP**.
+    
+    2. Expand the **Login Policies** -> **Account Locking** and select **Account Lock Enabled**.  
+        Once the user activates the account via the confirmation email, an **Account Unlocked** email 
+        will be sent by the Identity server.
+        
+        For more information aboutaccount locking, see [Account
```

Comment:
missing a space between "about" and "account"

Reply:
Fixed: 34640c47b5c2100d1c6d914bcc9b6aeb406df775